EIPS/EIP-1776.md

```diff
@@ -0,0 +1,187 @@
+---
+eip: 1776
```

Comment:
```suggestion
eip: 2981
```
Good catch @wighawag.  Please use 2981 for this EIP, and rename the file to `eip-2981.md` (note the casing).